integration-tests/tests/src/hooks/import-app-before.ts

```diff
@@ -0,0 +1,13 @@
+import { importApp, TemplateReplacements } from "../utils/import-app";
+
+export function importAppBefore(name: string, replacements: TemplateReplacements = {}, logLevel: Realm.App.Sync.LogLevel = "warn") {
+  before(async function(this: Partial<AppContext> & Mocha.Context) {
+    this.timeout(10000);
```

Comment:
Why the timeout?  Perhaps add a comment and maybe we can think of a way to conditionally await some sort of readiness.

Reply:
Oh, this is not calling `setTimeout` internally. This is just to prevent the hook from failing if it takes more than 2s (and less than 10s in this case) to import the app.

Reply:
A very good.  So this will return if it takes too long.  I'm happy with that :)

Reply:
Yeah, it will return (or basically abort / fail the suite) it the execution of the hook takes too much time (10s in this case).